Backend/templates/gitlab-ci/master-gitlab-ci.yml

```diff
@@ -25,27 +25,35 @@ stages:
 
 .build_mr_template:
   stage: MergeRequest
+  environment:
+    name: development
   only:
     - merge_requests
   
 
 .lint_mr_template:
   stage: MergeRequest
+  environment:
+    name: development
   only:
     - merge_requests
 
 
 LintHelmMR:
   stage: MergeRequest
   image: devth/helm
+  environment:
+    name: development
   only:
     - merge_requests
   script:
-    - helm lint $CHART_NAME
+    - helm lint ./chart/$NAME
```

Comment:
don't forget to change this when we finish all the helm specification

Reply:
done